app/build.gradle

```diff
@@ -64,6 +71,16 @@ dependencies {
     implementation 'androidx.navigation:navigation-fragment-ktx:2.5.1'
     implementation 'androidx.navigation:navigation-ui-ktx:2.5.1'
 
+    // Compose
+    implementation "androidx.compose.runtime:runtime:$rootProject.composeVersion"
```

Comment:
Very minor, but might help maintenance of the test project, all of these versions apart from accompanist can be grabbed from the Jetpack Compose BoM 👍  https://developer.android.com/jetpack/compose/setup#setup-compose

Reply:
Thanks. Will update this